src/Confluent.Kafka/Confluent.Kafka.csproj

```diff
@@ -1,6 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <Authors>Confluent Inc.;Andreas Heider</Authors>
+    <Description>Confluent's .NET Client for Apache Kafka</Description>
+    <Copyright>Copyright 2016-2017 Confluent Inc., Andreas Heider</Copyright>
+    <PackageTags>Kafka;Confluent;librdkafka</PackageTags>
```

Comment:
Only this csproj metadata are important - the others are not packaged so I dont think the metadata will be shown anywhere. Some important metadata missing here:

- Add` <PackageProjectUrl>` (https://github.com/confluentinc/confluent-kafka-dotnet/) It's the first thing I visit before installing a package. And `<RepositoryUrl>`
- Add `<PackageIconUrl>` (confluent or kafka? Or is there an icon with both? 64*64 with transparent background)
- Add in the description that it use librdkafka.
- Add the `<PackageLicenseUrl>` (given this, should we add all the licences listed on librdkafka in this repo?)
- For later, we will want to add `<PackageReleaseNotes>`. Always good to know what we upgrades directly in nuget package manager
- Add Summary and modify Description : in nuget package manager and nuget.org, Summary is the line just under the package name (so one of the most important), description is shown when you focus on this package. Title should be short, Description can be long. Example:
```xml
<Summary >.NET Client for Apache Kafka</Summary >
<Description>
Confluent.Kafka is a high-performance, reliable client for Apache Kafka

It is a wrapper around librdkafka, a finely tuned C client common to all other confluent kafka client, kept up to date with the latest addition in kafka protocol.
Maintained by Confluent, the company founded by the creators of Apache Kafka.
</Description>
```
**Edit**: I said a mistake first, this is not `Title` but `Summary` we want. And this one is not supported by msbuild yet. So we either have to create a .nuspec, either create the package without it (it will then truncate the description, so we may just have the first line in description to be what we want and this will be ok)